### PR TITLE
Removing filled radio style on focus

### PR DESCRIPTION
**Pull request by ryandialpad**
## Description

Removing confusing fill from radio button when it is focused

## Pull Request Checklist

 - [x] Ask the contributors if a similar effort is already in process or has been solved.
 - [x] Review the [contribution guidelines](https://github.com/dialpad/dialtone/blob/staging/.github/CONTRIBUTING.md).
 - [x] Use `staging` as your pull request's base branch. (All PRs using `production` as its base branch will be declined).
 - [x] Ensure all `gulp` scripts successfully compile.
 - [x] Update, remove, or extend all affected documentation.
 - [x] Provide a description what is being changed, extended, or removed.
 - [x] Link to any issue(s) this request is resolving.
 - [x] Ensure no private Dialpad links or info are in the code or pull request description (Dialtone is a public repo!).
 - [x] Request a review from Drew Chandler, Joshua Hynes, or Ted Goas.

## Screenshot
Screen Shot 2021-06-07 at 3 46 30 PM

### CODE REVIEW

File: lib/build/less/components/radio-checkbox.less [hunk 1]
```diff
@@ -234,7 +234,6 @@
 
         &:focus,
         &:checked:focus {
-            background-color: var(--primary-color);
```
Comment (ryandialpad) [inline lib/build/less/components/radio-checkbox.less:237]: @dbecherdialpad @dialpad-drew Here is the single line change made to remove the fill

**Review by dialpad-drew — APPROVED** [review]
Interesting that the first group of radios in both the deploy preview and in prod don't get focus. It jumps down to the link. 

Not a part of this, though.  🚢

**Comment by dialpad-drew** [comment]
> Interesting that the first group of radios in both the deploy preview and in prod don't get focus. It jumps down to the link.
> 
> Not a part of this, though. 🚢

Ahh, I presume it's because the checked radio is disabled and therefore doesn't get focus.

File: lib/build/less/components/radio-checkbox.less (hunk 1 above)
Comment (dbecherdialpad) [reply lib/build/less/components/radio-checkbox.less:237]: just to clarify: a focused _checked_ radio will still have the fill? (I assume that's still handled by the styles below)

**Review by dbecherdialpad — APPROVED** [review]

File: lib/build/less/components/radio-checkbox.less (hunk 1 above)
Comment (ryandialpad) [reply lib/build/less/components/radio-checkbox.less:237]: Yep 😄 

`&:checked` provides the correct style

**Comment by dbecherdialpad** [comment]
can we reference that this closes #343

**Pull request merged**